translations/dutch.json

```
@@ -500,6 +500,134 @@
 		"You are not ignoring help ticket notifications. Use /helpticket ignore to stop receiving notifications.": "",
 		"You will now receive help ticket notifications.": "",
 		"${target} does not have a ticket.": "",
-		"You deleted ${target}'s ticket.": ""
+		"You deleted ${target}'s ticket.": "",
+
+		"Server version: <b>${Chat.packageData.version}</b>": "",
+		"/mee - must not start with a letter or number": "",
+		"What?! How are you not more excited to battle?! Try /battle! to show me you're ready.": "",
+		"Access denied for custom avatar - make sure you're on the right account?": "",
+		"Invalid avatar.": "",
+		"Avatar changed to:": "",
+		"Artist: ": "",
+		"No one has PMed you yet.": "",
+		"You forgot the comma.": "",
+		"User ${this.targetUsername} not found. Did you misspell their name?": "",
+		"User ${this.targetUsername} is offline.": "",
+		"The user \"${this.targetUsername}\" was not found.": "",
+		"The room \"${target}\" was not found.": "",
+		"You do not have permission to invite people into this room.": "",
+		"This user is already in \"${targetRoom.title}\".": "",
+		"Setting status messages in /busy is no longer supported. Set a status using /status.": "",
+		"Setting status messages in /away is no longer supported. Set a status using /status.": "",
+		"User '${target}' not found.": "",
+		"${targetUser.name} does not have a status set.": "",
+		"${targetUser.name}'s status \"${targetUser.userMessage}\" was cleared by ${user.name}${reason ? ": "",
+		"You don't have a status message set.": "",
+		"You have cleared your status message.": "",
+		"This user has not played any ladder games yet.": "",
+		"W[TN: initial for Wins]": "",
+		"L[TN: initial for Losses]": "",
+		"You already have the temporary symbol '${group}'.": "",
+		"You must specify a valid group symbol.": "",
+		"You may only set a temporary symbol below your current rank.": "",
+		"Your temporary group symbol is now": "",
+		"Currently, you're viewing Pok\u00e9mon Showdown in ${Chat.languages.get(user.language || 'english')}.": "",
```

```suggestion
		"Currently, you're viewing Pokémon Showdown in ${language}.": "",
```

I'd make a temporary variable `language` for this, and also just use Unicode in the translation files. (I'd leave it escaped in the source files themselves, though.)